### ISSUE TITLE
Experimental Design List & Loop

### ISSUE THREAD

**Issue by thehastingsp** [department-of-veterans-affairs/vets-design-system-documentation#1187]
## What
One question per page pattern to evolve list and loop.

## Purpose
List and loop presents a stack of several calls to action that are tough to read, adds cognitive load, and present a difficult hierarchy of actions for screen readers as well as general users. I'm proposing one question per page to better guide users through the form, reduce cognitive load by asking one question at a time, incorporate conversational design, and reduce confusion. This pattern has been workshopped with @humancompanion-usds, @joshkimux and @Mottie to ensure viability, accessibility and feasibility. 

## Usage
In a Supplemental Claim, Veterans are asked to identify potentially multiple resources to identify evidence to support their claim. They will need to enter several locations, dates, and documents to support their claim. 

## Behavior
This behavior pattern occurs across claims and appeals, so we aim to improve several forms with this approach. The intended user behavior is challenging in the current view, especially on mobile. The intent is to de-clutter visually, guide, and organize decisions to support one decision at a time. This pattern is also intended to help users change their minds and easily edit one thing at a time and avoid a cluttered screen that is hard to parse for screen readers.  

## Examples
![Experimental List Loop As Is](https://user-images.githubusercontent.com/68615337/188946672-2b8add9c-9308-41fc-a4b0-10e5bbda5612.png)
![Experimental List Loop Proposed](https://user-images.githubusercontent.com/68615337/188946675-097a0011-0114-42b3-a5e5-3cfbacfce8df.png)
[Prototype of proposed](https://www.figma.com/proto/JveclTFbNNSmx4eiYwbZdZ/Supplemental-Claims?node-id=450%3A14578&scaling=min-zoom&page-id=450%3A12823&starting-point-node-id=450%3A14578)

## Accessibility
I worked closely with @joshkimux and @Mottie to walk through accessibility concerns with this pattern. We specifically paid attention to h1s, redundant questions, and hierarchy in the guidance.

## Guidance
This pattern has the potential to impact overall usability of VA.gov if we consider conversational design. The hypotheses we are testing could carry through to other forms and establish a plain language way of gathering necessary information from Veterans, reducing their cognitive burden, and building trust.  

## Research (optional)
We intend to test this pattern in an [upcoming usability test](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Supplemental-Claims/Research/MVP%20Usability%20Research/researchplan.md). 

## Code (optional)
Not in code yet. We are testing a prototype.

## Next steps
This has been workshopped in design system office hours and one-on-ones, so we will aim to bring our usability findings to the design council.

**Comment by caw310** [comment]
This will be on the DSC agenda for the 9/16 meeting.

**Comment by caw310** [comment]
Follow up from DSC meeting. 
We will document pattern and mark it as new and indicate that some testing was done but that we need more testing on this. 
Will add to DST backlog to document this pattern and other list and loop patterns Robin has identified.

**Comment by thehastingsp** [comment]
thanks @caw310 are there any specific follow up actions for me?

**Comment by caw310** [comment]
Not at this time. When we add the pattern, Natalie or Matt will probably reach out to you to see assets. Thanks!

**Comment by humancompanion-usds** [comment]
We've codified and documented a multi-page list and loop in issue #2822.
